### PR TITLE
replaced gentx

### PR DESCRIPTION
**Pull request by hannydevelop**

### CODE REVIEW

File: grenache/gentx/Ugochi.json [hunk 1]
```diff
@@ -32,10 +32,10 @@
                 "validator_address": "sommvaloper1te5l2m7vcttph49crmtldmhxysscqnpy26g9rk",
                 "orchestrator_address": "somm14xdxwtd2wu0flcma2qf7g2c6zx59dasuazddu6",
                 "ethereum_address": "0x5d5A6DfCEcc822FdeD4493A449C1e2Da7F08Bd09",
-                "eth_signature": "4CXC8z8VqI77zkDzwX6+7Y8tv2d6EodyM2AzuUaYkJJkIoZMvufQ6Oq57060L3kCEob63yHbo4mNxJbQ4Oyuqxs="
```
Comment (EricBolten) [inline grenache/gentx/Ugochi.json:35]: Can you re-run the commands to check your validator/orchestrator/ethereum addresses? It is unexpected for your `eth_signature` to change given the same values.

**Review by EricBolten — COMMENTED** [review]
Need to double check accuracy.

File: grenache/gentx/Ugochi.json (hunk 1 above)
Comment (hannydevelop) [reply grenache/gentx/Ugochi.json:35]:
```
root@grenache-ugochi:~# gorc --config ~/gorc/config.toml keys cosmos show orchestrator
orchestrator	somm14xdxwtd2wu0flcma2qf7g2c6zx59dasuazddu6
root@grenache-ugochi:~# gorc --config ~/gorc/config.toml keys eth show signer
0x5d5A6DfCEcc822FdeD4493A449C1e2Da7F08Bd09
root@grenache-ugochi:~# sommelier keys show validator --keyring-backend test
- name: validator
  type: local
  address: somm1te5l2m7vcttph49crmtldmhxysscqnpylyxcye
  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AmoJO5F32mMyuavv7lQC5HCU19l+hjE+NgzU7+2y4n34"}'
  mnemonic: ""
```

Comment (hannydevelop) [reply grenache/gentx/Ugochi.json:35]:
```
root@grenache-ugochi:~# cat /root/.sommelier/config/gentx/gentx-b599397a5644357f3179d358d98bf943fbd5bf44.json | jq
{
  "body": {
    "messages": [
      {
        "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
        "description": {
          "moniker": "moniker",
          "identity": "",
          "website": "",
          "security_contact": "",
          "details": ""
        },
        "commission": {
          "rate": "0.100000000000000000",
          "max_rate": "0.200000000000000000",
          "max_change_rate": "0.010000000000000000"
        },
        "min_self_delegation": "1",
        "delegator_address": "somm1te5l2m7vcttph49crmtldmhxysscqnpylyxcye",
        "validator_address": "sommvaloper1te5l2m7vcttph49crmtldmhxysscqnpy26g9rk",
        "pubkey": {
          "@type": "/cosmos.crypto.ed25519.PubKey",
          "key": "5vrvRwF/7Th07Cjm0q+4y7z8zKTXeRwJDh013lm4qbg="
        },
        "value": {
          "denom": "usomm",
          "amount": "1000000000"
        }
      },
      {
        "@type": "/gravity.v1.MsgDelegateKeys",
        "validator_address": "sommvaloper1te5l2m7vcttph49crmtldmhxysscqnpy26g9rk",
        "orchestrator_address": "somm14xdxwtd2wu0flcma2qf7g2c6zx59dasuazddu6",
        "ethereum_address": "0x5d5A6DfCEcc822FdeD4493A449C1e2Da7F08Bd09",
        "eth_signature": "K9ejVmWT0Ileljl31EBSCGU/CCswpbZcVbIg0dp8s5ln0YJoTXqXY5RM7YTzPl7DB9bKbllUbGnATXoP9JZs3Rs="
      }
    ],
    "memo": "dc62b101b65c868d930fa18345bdb4870dc14e98@192.168.43.13:26656",
    "timeout_height": "0",
    "extension_options": [],
    "non_critical_extension_options": []
  },
  "auth_info": {
    "signer_infos": [
      {
        "public_key": {
          "@type": "/cosmos.crypto.secp256k1.PubKey",
          "key": "AmoJO5F32mMyuavv7lQC5HCU19l+hjE+NgzU7+2y4n34"
        },
        "mode_info": {
          "single": {
            "mode": "SIGN_MODE_DIRECT"
          }
        },
        "sequence": "0"
      }
    ],
    "fee": {
      "amount": [],
      "gas_limit": "200000",
      "payer": "",
      "granter": ""
    }
  },
  "signatures": [
    "CfnC9Hdb47etrKZdbNRzauki9EOCRUypDs8/M5W4Cc5xMJ31FVCyb3MWqErpTTdxbFpdWvZNrzxIfVrdCqTShQ=="
  ]
}
root@grenache-ugochi:~# 
```

**Comment by EricBolten** [comment]
Hrmmm, if you regenerated the delegate signature to run this gentx, then the old one must have had the wrong value. Good thing we updated it then!

**Review by EricBolten — APPROVED** [review]
LGTM

File: grenache/gentx/Ugochi.json [hunk 2]
```diff
@@ -32,10 +32,10 @@
...
+                "eth_signature": "K9ejVmWT0Ileljl31EBSCGU/CCswpbZcVbIg0dp8s5ln0YJoTXqXY5RM7YTzPl7DB9bKbllUbGnATXoP9JZs3Rs="
             }
         ],
-        "memo": "",
+        "memo": "dc62b101b65c868d930fa18345bdb4870dc14e98@192.168.43.13:26656",
```
Comment (EricBolten) [inline grenache/gentx/Ugochi.json]: Whoops, I didn't notice this until looking at it again, this IP address is not your externally facing one, it's local. Also this node ID doesn't match the one in your addresses.json file. One or the other needs to be updated to reflect what is currently accurate on your machine.

**Review by EricBolten — CHANGES_REQUESTED** [review]
Noticed an issue that needs fixing.

File: grenache/gentx/Ugochi.json (hunk 2 above)
Comment (EricBolten) [reply grenache/gentx/Ugochi.json]: I think the values in addresses.json are the correct ones.

**Review by EricBolten — APPROVED** [review]
LGTM

**Pull request merged**